airflow/config_templates/config.yml

```diff
@@ -452,6 +452,14 @@ core:
 database:
   description: ~
   options:
+    alembic_ini_file_path:
+      description: |
+        Optional path to a custom alembic.ini file. If provided, these configurations
+        will be prioritised over the default alembic.ini file.
+      version_added: 2.7.0
+      type: string
+      example: ~
+      default: ~
```

Comment:
We can do that by adding the default value as `alembic.ini`

Reply:
I’d say

```ini
alembic_ini_file_path = alembic.ini
```

implies prepending `airflow.__file__` (which is added in Python code), and if you don’t want that to be prepended you need something like

```ini
alembic_ini_file_path = /path/to/my/config/file.ini
```

This can be detected with `os.path.isabs`.

Reply:
Thanks for the feedback. Pushed a latest commit for that